packages/kit/src/core/adapt/prerender.js

```diff
@@ -5,6 +5,12 @@ import { mkdirp } from '../filesystem/index.js';
 import { __fetch_polyfill } from '../../install-fetch.js';
 import { SVELTE_KIT } from '../constants.js';
 
+/**
+ * @typedef {import('types/config').PrerenderErrorHandler} PrerenderErrorHandler
+ * @typedef {import('types/config').ValidatedConfig['kit']['prerender']['onError']} OnErrorConfig
```

Comment:
I think a nicer way to do this line would be to create a value in the `.d.ts` file that's exported from there and then imported here:

`export type OnErrorValue = 'fail' | 'continue' | PrerenderErrorHandler`

Reply:
Agreed. Basically every time I try to not presume anything, and just work with what is available, I find I just should have made the slightly bigger change. 😆